countrycodepicker/src/main/res/values-pt/strings_country.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="country_afghanistan_name">Afeganistão</string>
```

Comment:
I think this line takes care of showing the country names in a given language: https://github.com/joielechong/CountryCodePicker/blob/master/countrycodepicker/src/main/java/com/rilixtech/widget/countrycodepicker/CountryCodeArrayAdapter.java#L78

Also see: https://developer.android.com/reference/java/util/Locale#getDisplayCountry()

So we don't need to add `strings_country.xml` files for any language